src/main/java/org/apache/commons/cli/HelpFormatter.java

```diff
@@ -129,6 +129,14 @@ public Builder setShowDeprecated(final boolean useDefaultFormat) {
         }
     }
 
+    /**
+     * Returns the option description or an empty string if the description is {@code null}.
+     * @param option The option to get the description from.
+     * @return the option description or an empty string if the description is {@code null}.
+     */
```

Comment:
- New public and protected elements need Javadoc `since` tags.
- In Javadoc, a foo getter methods "Gets foo..." (I've tried to be consistent with this in Commons.)